### ISSUE TITLE
Login Loop

### ISSUE THREAD

**Issue by jclrc** [nextcloud/docker#1530]
Just found that Netxcloud 22 was available and setup a test site using this docker image.  I grabbed a sample docker-compose.yml file from .examples/docker-compose/insecure/mariadb/apache/docker-compose.yml and made no changes other than passwords.  After bringing it up I find that I get stuck in a login loop.  It appears that after login the server is attempting to redirect me to the dashboard because I get the URL of
login?redirect_url=/apps/dashboard/ but I stay on the login screen.

I have searched and found a person on Nextcloud 20 who had the same issue but was not using a docker image.  His fix was to create a missing folder.  Anyone else having this issue with this docker image?  I can provide my docker-compose file, but it is the same as the example I list above.

Thanks!

**Comment by JuniorJPDJ** [comment]
Do you have redis enabled here?

**Comment by jclrc** [comment]
Yes

Thank You,
John Curtis


On Jul 9, 2021, at 22:16, JuniorJPDJ ***@***.***> wrote:

﻿

Do you have redis enabled here?

**Comment by JuniorJPDJ** [comment]
Try disabling redis, there were problems with it.

**Comment by jclrc** [comment]
> Try disabling redis, there were problems with it.

Thanks JuniorJPDJ, but I started from scratch and did not include redis and ended up with the same login loop problem.
I also downloaded Netxcloud 22 and installed it in a Linux container with Mariadb and had no problems.  I would really rather use a docker container for this purpose.  I was hoping not to have to write my own image.

Any other thoughts?
